rules/windows/execution_ms_office_written_file.toml

```diff
@@ -24,14 +24,14 @@ type = "eql"
 query = '''
 sequence with maxspan=2h
   [file where event.type != "delete" and file.extension == "exe" and
-     process.name in ("winword.exe",
-                      "excel.exe",
-                      "outlook.exe",
-                      "powerpnt.exe",
+     process.name in ("WINWORD.EXE",
```

Comment:
an `x : ... or x : ... or ...` is the only way to do a case-insensitive `in` for 7.10.

if we don't have time to be complete everywhere and just need to get this through, that's okay. we can pick it up when we have cycles (and aren't both moving)